routes/offer.js

```diff
@@ -13,6 +13,7 @@ router.use(authMiddleware)
 router.param('id', idValidation)
 
 router.get('/', asyncWrapper(offerController.getOffers))
+router.get('/price-values', setCurrentUserIdAndRole, asyncWrapper(offerController.priceMinMax))
```

Comment:
rename endpoint as well
prolly price-range fits here better after all
price-min-max
doesnt look ok, does it?

Reply:
why do you use here setCurrentUserIdAndRole?
i doubt that you need the current user id for this endpoint, do you?

Reply:
fixed